src/site/content/en/blog/inp-in-frameworks/index.md

```diff
@@ -0,0 +1,236 @@
+
+<div class="table-wrapper scrollbar">
+<table>
+  <thead>
+    <tr>
+      <th></th>
+      <th>FID</th>
+      <th>INP</th>
+    </tr>
+  </thead>
+  <tbody>
+  <tr>
+   <th>Measurement</th>
+   <td>Measures the duration between the first user input and the time when the corresponding event handler runs.
+   </td>
+   <td>Measures the overall interaction latency by using the delay of the 
+    <ul>
+      <li>single largest interaction for less than 50 transactions 
+      <li><a href="https://web.dev/inp/#why-not-the-worst-interaction-latency">one of the largest interactions</a> for more than 50 transactions.</li>
+    </ul>
+   </td>
+  </tr>
+  <tr>
+   <th>Depends on</th>
+   <td>Main thread availability to run the event handler required for the first interaction. The main thread could be blocked because it is processing other resources as part of the initial page load.
+   </td>
+   <td>Main thread availability and size of the script executed by the event handlers for different interactions, including the first interaction.
+   </td>
+  </tr>
+  <tr>
+   <th>Primary cause for poor scores</th>
+   <td>Poor FID is mainly caused due to <a href="https://web.dev/optimize-fid/#heavy-javascript-execution">heavy JavaScript execution</a> on the main thread.
+   </td>
+   <td>Heavy event-handling JavaScript and other rendering tasks after running handlers can result in poor INP.
+   </td>
+  </tr>
+  <tr>
+   <th>Optimization</th>
+   <td>FID can be optimized by improving resource loading on page load and optimizing JavaScript code.
+   </td>
+   <td>Similar to FID for every interaction plus usage of rendering patterns that prioritize key UX updates over other rendering tasks. 
+   </td>
+  </tr>
+   </tbody>
+   <caption>FID versus INP: Measurement and optimization</caption>
+</table>
+</div>
+
+The [Aurora](/introducing-aurora/) team in Chrome works with open-source web frameworks to help developers improve different aspects of the user experience, including performance and CWV metrics. With the introduction of INP, we want to be prepared for the change in CWV metrics for framework-based websites. We have collected data based on the experimental responsiveness metric in CrUX reports. We will share insights and action items to ease the transition to the INP metric for framework-based websites. 
+
+
+## Experimental responsiveness metric data
+
+An INP below or equal to 200 milliseconds indicates good responsiveness. The CrUX report data and the [CWV Technology Report](https://www.google.com/url?q=https://datastudio.google.com/s/gPxRJmoivLA&sa=D&source=docs&ust=1652475737927418&usg=AOvVaw3ZQ0WOZu2ZGjCuJb23dQYg) for April 2022 give us the following information about responsiveness for popular JavaScript frameworks.
+
+Note: This measurement includes data from all versions of the frameworks listed. For more mature frameworks, this can include data from versions many years out-of-date.
```

Comment:
```suggestion
{% Aside %}
This measurement includes data from all versions of the frameworks listed. For more mature frameworks, this can include data from versions many years out-of-date.
{% endAside %}
```